library/crates/Cargo.toml

```diff
@@ -70,19 +75,22 @@ genmode = "Remote"
 [package.metadata.raze.crates.axum.'0.5.15']
 compile_data_attr = "glob([\"**/*.md\"])"
 
+[package.metadata.raze.crates.axum-server.'0.4.4']
```

Comment:
Make sure the `*.der` files from `axum-server` crate gets included in the Bazel package generated by Cargo Raze